ROADMAP.md

```diff
@@ -2,40 +2,47 @@
 
 ## User facing roadmap
 
-### V1beta1 support and GA
+### pipeline API v1 support
 
-- Add support for v1beta1 and later on GA
-- Warn usage of PipelineResource in v1beta1 (no error but warn that it is not supported aka will work but can be broken in a later release)
-- Support for workspace, result, …
+- Add support for v1 API when it will be publish
+- Support for fields, …
 
-### Improve e2e Testing Coverage for CLI
+### triggers API v1beta1 support
 
-- We should do an assessment of what scenarios we have covered for e2e testing and figure out where additional testing is needed
+- Add support for v1beta1 triggers API
 
-### Consistency across all CLI resources for list, delete, and describe subcommands
+### results integration
 
-- We should work to make sure each resource introduced to the CLI always features these three subcommands
+Evaluate and provide an integration with the
+[results](https://github.com/tektoncd/results) project.
 
-### Filtering for list and delete commands
+### chains integration
 
-- We should provide filtering solutions for all list/delete commands associated with CLI
+Evaluate and provide an integration with the
+[chains](https://github.com/tektoncd/chains) project.
+
+### operator integration
+
+Evaluate and provide an integration with the
```

Comment:
Do we have a CLI integration in mind for the operator?

Reply:
related to this, i think it would be nice if tkn was able to bootstrap tekton and components,

i.e: 

`tkn bootstrap pipeline`

will grab pipeline latest release.yaml and apply it

`tkn bootstrap operator`

will install the operator

and so on for other components.....